Userland/Libraries/LibWeb/HTML/HTMLMediaElement.cpp

```diff
@@ -435,10 +437,178 @@ enum class SelectMode {
     Children,
 };
 
+class SourceElementSelector final : public Bindings::PlatformObject {
+    WEB_PLATFORM_OBJECT(SourceElementSelector, Bindings::PlatformObject);
```

Comment:
Does this actually need to be a `PlatformObject`? Since it's not exposed to the web, can we use a plain `JS::Cell` instead?